### PR TITLE
Small updates to README

### PR DESCRIPTION
**Pull request by leina05**
Not sure if these are really required. After re-reading the README, I think it provides enough information to users, without repeating information from the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,18 @@ authorization logic. You define this logic separately from the rest of your
 application code, but it executes inside the application and can call
 directly into it. oso ships as a library with a built-in debugger and REPL.
 
+oso is ideal for building permissions into user-facing applications, but you can
+check out [Use Cases](https://docs.oso.dev/more/use-cases.html) to learn about
+other applications for oso.
+
+Using oso basically consists of two parts:
```
Comment (samscott89) [inline README.md:21]:
```suggestion
Using oso consists of two parts:
```

File: README.md [hunk 2]
```diff
@@ -14,10 +14,18 @@ authorization logic. You define this logic separately from the rest of your
 application code, but it executes inside the application and can call
 directly into it. oso ships as a library with a built-in debugger and REPL.
 
+oso is ideal for building permissions into user-facing applications, but you can
+check out [Use Cases](https://docs.oso.dev/more/use-cases.html) to learn about
```
Comment (gj) [inline README.md:18]: docs.osohq.com

File: README.md [hunk 3]
```diff
@@ -14,10 +14,18 @@ authorization logic. You define this logic separately from the rest of your
...
+other applications for oso.
+
+Using oso consists of two parts:
+
+1. Writing oso policies in a declarative policy language called Polar
+2. Embedding oso in your application using the appropriate language-specific authorization library
```
Comment (gj) [inline README.md:24]:
```suggestion
1. Writing oso policies in a declarative policy language called Polar.
2. Embedding oso in your application using the appropriate language-specific authorization library.
```

File: README.md [hunk 4]
```diff
@@ -14,10 +14,18 @@ authorization logic. You define this logic separately from the rest of your
...
+
 ## Getting started
 
-To get up and running with oso, check out the [Getting Started guide][docs] in
-the [oso documentation][docs].
+To get up and running with oso, check out the [Getting Started guides](https://docs.oso.dev/getting-started/quickstart.html) in the [oso documentation][docs].
```
Comment (gj) [inline README.md:28]: docs.osohq.com

File: README.md (hunk 3 above)
Comment (leina05) [reply README.md:24]: Not sure we need the period since these aren't full sentences

Comment (gj) [reply README.md:24]: works for me

**Pull request merged**